src/components/select.css

```diff
@@ -0,0 +1,20 @@
+.modal-content {
+
+  /* border:solid tomato; */
+margin: 0rem auto;
+
+}
+
+img {
+
+width:350px;
+height:350px;
+
+}
+
+.modal-header {
+
+flex-direction: column;
+
+
+} 
```

Comment:
Really lovely work!

Only feedback here is regarding spacing, try and clean this up before the next merge 👍